internal/ping/ping.go

```diff
@@ -0,0 +1,38 @@
+package ping
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/influxdata/influx-cli/v2/api"
+	"github.com/influxdata/influx-cli/v2/pkg/tracing"
+)
+
+type Client interface {
+	GetHealth(context.Context, *api.GetHealthParams, ...api.RequestEditorFn) (*http.Response, error)
+}
+
+// Ping checks the health of a remote InfluxDB instance.
+func Ping(ctx tracing.Context, client Client) error {
+	resp, err := client.GetHealth(ctx, &api.GetHealthParams{ZapTraceSpan: ctx.TraceId()})
+	if err != nil {
+		return fmt.Errorf("failed to make health check request: %w", err)
+	}
+	parsed, err := api.ParseGetHealthResponse(resp)
+	if err != nil {
+		return fmt.Errorf("failed to parse health check response: %w", err)
+	}
+
+	if parsed.StatusCode()/100 != 2 {
+		return fmt.Errorf("health check failed (got status %d): %s", parsed.StatusCode(), string(parsed.Body))
+	}
+	if parsed.JSON200 == nil {
```

Comment:
The codegen picks weird names for things, I think because we haven't set an `operationId` on the response schemas(?)